elasticdl/python/elasticdl/feature_column/feature_column.py

```diff
@@ -210,7 +210,80 @@ def embedding_weight_name(self):
         return self._embedding_delegate.name
 
 
-def concat_column(categorical_columns):
+def concatenated_categorical_column(categorical_columns):
+    """A `CategoricalColumn` to concatenate multiple categorical columns.
+
+    Use this when you have many categorical columns and want to map all
+    to embedding. It's recommended to concatenate the sparse id tensors
+    from the categorical columns above to one sparse id tensor using this
+    API, and then map the combined SparseTensor to Embedding. Because
+    there is much overhead to create embedding variables for each
+    categorical column, the model size will be huge. Using this way, we
+    can reduce the model size.
+
+    The id range of source categorical columns are [0, num_buckets_0],
```

Comment:
"The id" -> "The integer id"

Reply:
Why is it `[0, num_buckets_0]` not `[0, num_buckets_0)`

Reply:
> Why is it `[0, num_buckets_0]` not `[0, num_buckets_0)`

Good catch, will update it.

Reply:
> "The id range of " -> "The output integer id range of"

Updated to `The output id range of`